app/utilities/schema.py

```diff
@@ -13,7 +14,7 @@
 
 logger = get_logger()
 
-DEFAULT_SCHEMA_DIR = "data"
+SCHEMA_DIRS = ["schemas", "test_schemas"]
```

Comment:
This doesn't need to be a module level variable now. It's only used once in get_schema_path_map_for_language

Reply:
I think there's value keeping it there in order to keep configuration of schema behaviour together. The same could be said for the other module level variables in this file.